hiera.yaml

```diff
@@ -0,0 +1,16 @@
+---
+version: 4
+datadir: data
+hierarchy:
+  - name: 'Full version'
+    backend: yaml
+    path: "%{facts.os.name}-%{facts.os.release.full}"
```

Comment:
Ok, maybe we should do it like puppetlabs-ntp, with Full version, Major version, Distro name, OS family and common? https://github.com/puppetlabs/puppetlabs-ntp/blob/master/hiera.yaml

And are you ok with bumping the stdlib version requirement or should I remove the `Stdlib::Absolutepath` type again?